### PR TITLE
Fix/clean data date

### PR DESCRIPTION
**Pull request by saberg1**
## Description

Add functionality for post request to send post to api. Create variable to handle data cleaning for date. Remove comments

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor

## Checklist:
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/components/Form/Form.js [hunk 1]
```diff
@@ -20,7 +20,11 @@ const Form = ({ handleAddTip }) => {
 
     const onAddTip = (e) => {
         e.preventDefault();
-        handleAddTip({ title: title, description: description, mod: mod, upvotes: 0, date: Date.now() })
+
+        const date = new Date(Date.now());
```
Comment (tylrs) [inline src/components/Form/Form.js]: Maybe we could do .tolocalestring() after newDate(Date.now())

**Review by tylrs — CHANGES_REQUESTED** [review]
revise the date.now syntax

File: src/components/Form/Form.js (hunk 1 above)
Comment (saberg1) [reply src/components/Form/Form.js]: Done and Done

Comment (tylrs) [reply src/components/Form/Form.js]: Much better

**Review by tylrs — APPROVED** [review]

**Pull request merged**